src/data/enums/AIModel.ts

```diff
@@ -1,4 +1,5 @@
 export enum AIModel {
     OBJECT_DETECTION = "OBJECT_DETECTION",
-    POSE_DETECTION = "POSE_DETECTION"
+    POSE_DETECTION = "POSE_DETECTION",
+    OBJECT_DETECTION_YOLOv5 = "OBJECT_DETECTION_YOLOv5"
 }
```

Comment:
Probably Object_Detection and Pose Detection should have a separate enum  to chose between SSD, YOLO, etc.